### ISSUE TITLE
[Accessibility] [Time Picker > Clear Value] : Status message is not annnounced when value is cleared.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#2092]
[A11Y-572](https://jira.cengage.com/browse/A11Y-572)
[A11Y-583](https://jira.cengage.com/browse/A11Y-583)

**Issue 1:**
**[Accessibility] [Time Picker> Custom Styles]: Color contrast ratio is not sufficient for the text 'Helper Text'.**

Platform:
Windows 11 / Chrome

Steps to reproduce:
1. Open the URL: https://react-magma.cengage.com/version/4.9.1
2. Now Navigate to side navigation and expand 'Components' button.
3. Now activate 'Time Picker' link under API
4. Now navigate to Custom styles section and verify.

Actual Result:
The text ''Helper text” fails to meet the minimum contrast ratio requirement of 4.5:1 for normal text.

Expected Result:
The text should meet or exceed the minimum contrast ratio of 4.5:1 against the background, ensuring readability for users with visual impairments.

WCAG Guideline:
1.4.3 Contrast (Minimum) Level AA

User Impact:
Users with low vision or color deficiencies may struggle to read or recognize the group label, impacting form comprehension and usability.

Recommendation:
Update the text color of the legend to ensure it meets the minimum contrast requirement.

Screencast:
https://app.screencast.com/qD5Qqij8mZc4d

**Issue 2:**
**[Accessibility] [Time Picker > Clear Value] : Status message is not annnounced when value is cleared.**

Environment:
Windows 11/Chrome 137/ JAWS 2025
Windows 11/Edge 136/ NVDA 2025
Mac 15/Safari/ VoiceOver

Steps to Reproduce:
1. Open the URL: https://react-magma.cengage.com/version/4.9.1/api/time-picker/
2. Turn On screen reader (JAWS/NVDA).
2. Launch 'Time Picker' component page from left navigation.
4. On the 'Time Picker' page, under 'Clear Value', navigate to 'Clear Time' button through Tab key and press Enter to activate it.
5. Verify that no information is announced when time value is cleared.

Actual Result:
No information is announced by assistive technology when time picker value is cleared on activating 'Clear Time' control.

Refer to screencast:
https://app.screencast.com/hz2mSPU7EnrDn?conversation=f2eGV0KyLCtTpuHfkS2XvH

Expected Result:
When time picker value is cleared through 'Clear Time' control, information like "time has been cleared" should be announced by assistive technology.

User Impact:
Screen reader users will not be able to identify that the value is cleared if it will not be announced to them.

WCAG Guideline:
4.1.3 Status Messages (AA)

Bug Fix Recommendation:
Add a hidden live region and update it with the message "Time has been cleared" when 'Clear Time' control is triggered.

**Not an issue with the component, but the doc example can be updated to showcase best practices.**

**Comment by silvalaura** [comment]
DOCS ISSUES (low priority)

Issue 1: pick a different background color for the custom styles example
Issue 2: Add Announce when Clear Time button is clicked on the docs

**Comment by nikitaorliak-cengage** [comment]
**Tested on:**
- V5: https://rc--upbeat-sinoussi-f675aa.netlify.app/api/time-picker/#clear_value
- V4: https://next--upbeat-sinoussi-f675aa.netlify.app/api/time-picker/#clear_value

Works as expected. No bugs found.
